### ISSUE TITLE
Click anywhere to upload

### ISSUE THREAD

**Issue by cavis** [PRX/publish.prx.org#109]
In story edit, click anywhere on audio file template to initiate upload

**Comment by gcampo88** [comment]
Currently on staging, if you click near "Upload a file to get started" (on a new story without any templates), you get an error: `Uncaught TypeError Cannot read property 'click' of undefined`. Looks like it's coming from the `(click)="upinput.click()"` in upload.component.ts:27.

**Comment by sandikbarr** [comment]
I don't think I understand the "why" behind this requirement. I think that I understand the result to be, upload an additional file. 

I thought from Genevieve's user feedback we were going to have an upload button for each file and allow changing the existing audio files? Is this requirement related to that feedback or something else?

**Comment by kookster** [comment]
yes, there are distinct Upload links per segment, but there is alot of
screen real estate for each segment that is basically empty and doing
nothing until you upload a file, so  I think the idea was you should be
able to click anywhere on a segment row/div that has no audio file yet and
have that work like hitting the upload button. Kinda like on a video player
where you can hit anywhere on the video window to start it playing, not
just the triangle in the middle.

**Comment by gcampo88** [comment]
My understanding of #109 was that this would just be a little more user-friendly user upload-- you could add a new file by clicking anywhere near the "Upload File" button (particularly on a standalone story, where all you see at first is that big "Upload a file to get started" `h4`). Currently on staging, looks like there was an attempt to make that happen with that `h4`, but it gives you an error: `Cannot read property 'click' of undefined`. 
So basically I'm just trying to make the clicks of anywhere in the Audio Files section trigger an upload-- but ignoring clicks that are on previously uploaded files, since I assume the user isn't necessarily trying to upload a new file when they click on the one they just uploaded. 
All sound ok?

**Comment by sandikbarr** [comment]
Ok, so isn't related to G's feedback. This all makes sense. Thanks.
